src/main/java/org/dcsa/ovs/model/enums/EventType.java

```diff
@@ -3,6 +3,7 @@
 public enum EventType {
     SHIPMENT,
```

Comment:
All this code will probably be moved to a Event_Core library - but OVS does not need SHIPMENT og EQUIPMENT events?

Reply:
As discussed we wait for the library to come